readme.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/kchitalia/SalesforceMobileSDK-iOS.svg?branch=unstable)](https://travis-ci.org/kchitalia/SalesforceMobileSDK-iOS)
```

Comment:
Shouldn't this be ```https://travis-ci.org/forcedotcom/SalesforceMobileSDK-iOS```? We should use the main repo. Also, could we move this to the bottom of this file?